cityofaustin/atd-moped
# 15855 handle activity log when parent deleted

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/15855

This bug appears to have been related to the way Javascript handles null objects, which is interesting! This fix should hopefully prevent this from popping up again.

## Testing
**URL to test:** 

[deploy preview](https://app.netlify.com/sites/atd-moped-main/deploys/662958dba988a100088cbc35)

**Steps to test:**
- go to a project and add a subproject to it
- remove the subproject
- navigate to the subproject you just removed
- the activity log should say a parent project was removed with the id of the parent project
- try adding/removing other fields from a project and checking the removal statement to make sure it makes sense
- only fields with lookup values should display lookup values, number and id fields should display raw data

here are some examples!
<img width="456" alt="Screenshot 2024-04-30 at 7 59 34 PM" src="https://github.com/cityofaustin/atd-moped/assets/35410637/86df4a8b-bcbd-43d0-ba34-aefb7d8599fb">


---
#### Ship list
